cmd/neofs-cli/modules/control/healthcheck.go

```diff
@@ -60,8 +61,17 @@ func healthCheck(cmd *cobra.Command, _ []string) {
 
 	verifyResponse(cmd, resp.GetSignature(), resp.GetBody())
 
+	healthStatus := resp.GetBody().GetHealthStatus()
+
 	cmd.Printf("Network status: %s\n", resp.GetBody().GetNetmapStatus())
-	cmd.Printf("Health status: %s\n", resp.GetBody().GetHealthStatus())
+	cmd.Printf("Health status: %s\n", healthStatus)
+
+	switch healthStatus {
```

Comment:
Can be simplified, we don't have many options now.

Reply:
find `switch` way more explicit for a reader but dont mind